File: app/src/main/java/com/alim/greennote/di/Note.java
package com.alim.greennote.di;

import android.app.Application;

import com.nelu.ncbase.BaseNC;

public class Note extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Injection.INSTANCE.init(this);
        BaseNC.INSTANCE.init(
                this, "98a1c01e-e4a4-49a1-bb08-21352a4602ee", null
        );
    }
}

File: app/src/main/java/com/alim/greennote/di/Note.kt
package com.alim.greennote.di

import android.app.Application
import com.nelu.ncbase.BaseNC.init

class Note : Application() {

    override fun onCreate() {
        super.onCreate()
        Injection.init(this)
        init(
            this, "98a1c01e-e4a4-49a1-bb08-21352a4602ee", null
        )
    }
}
